ensime.el

```diff
@@ -49,6 +49,7 @@
 (require 'ensime-stacktrace)
 (require 'ensime-debug)
 (require 'ensime-editor)
+(require 'ensime-expand-region)
```

Comment:
this is adding a hard dependency, it'll fail if `expand-region` is not installed.

Rather add a hard dependency to `(require 'expand-region)` in `ensime-expand-region.el`, and then let users `(require 'ensime-expand-region)` if they want this.